docs/Child-friendly-EmulationStation.md

```diff
@@ -13,23 +13,23 @@ These modes allow administrators to lock down (part of) the system, so that naiv
 ***
 
 # UI modes
-To allow different levels of system protection, user modes were introduced with different sets of functionalities. These modes are Full, Kiosk and Kid. Generally speaking, kids do not need to edit system settings, nor will visiting friends. The full set of options also complicates the UI, and the kid and kiosk modes hide these when they are not needed.
+To allow different levels of system protection, user modes have been introduced to allow the user to restrict certain settings and files from use. The three new modes are Full, Kiosk and Kid. In general, visiting friends and children do not need to modify system settings. The full set of options also complicates the UI, and the kid and kiosk modes hide these for a simpler experience.
 
 Setting UI mode to Full will give you the default ES user interface. This includes options such as CONFIGURE INPUT, UI SETTINGS, SCRAPER, but also EDIT THIS GAMES METADATA, and many other menu items that allow you to configure the system.
 
-Kiosk and Kid mode expose fewer of the menu options of Emulation station, only showing menu items that do not affect the system configuration (i.e. do not survive a reboot), with the exception of setting favorites. In Kiosk Mode, the user is still able to toggle the favorite status of a game. In Kid-mode, even this functionality is removed.
+Kiosk and Kid mode expose fewer of the menu options of Emulation Station, only showing menu items that do not affect the system configuration (i.e. do not survive a reboot). In Kiosk Mode, the user is still able to toggle the favorite status of a game. In Kid-mode, this functionality is removed.
```

Comment:
_Kid-mode_ should be _Kid mode_, for consistency.

Reply:
Resolved in 96f3dbbcb02027848cc006c951f7c0a5e8526ad9